.github/CONTRIBUTING.md

```diff
@@ -6,7 +6,11 @@ I try to manage the required tasks for MultiQC using GitHub issues, you probably
 
 However, don't be put off by this template - other more general issues and suggestions are welcome! Contributions to the code are even more welcome ;)
 
+<<<<<<< HEAD
+> _If you need help using MultiQC then the best place to go is the Gitter chatroom where you can ask me questions directly: https://gitter.im/ewels/MultiQC_
+=======
 > _If you need help using MultiQC then the best place to go is the Gitter chatroom where you can ask me questions directly: [https://gitter.im/ewels/MultiQC](https://gitter.im/ewels/MultiQC)_
+>>>>>>> upstream/master
```

Comment:
Whenever you see `<<<<<<< HEAD`, `=======` and `>>>>>>> upstream/master`, this is a git merge conflict marker that has been incorrectly committed instead of being resolved.

These need to be removed - see https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

Have a look at the diff in the _"Files changed"_ tab to see what code is being changed and make sure that it's only the stuff that you intend to be adding.

Let me know if you need any help!